core/new-gui/src/app/workspace/component/workflow-editor/workflow-editor.component.ts

```diff
@@ -659,10 +659,12 @@ export class WorkflowEditorComponent implements AfterViewInit {
 
   /**
    * Deletes the currently highlighted operators when user presses the delete key.
+   * When the delete key is clicked to INPUT or TEXTAREA fields, operator should not be deleted
    */
   private handleOperatorDelete() {
     Observable.fromEvent<KeyboardEvent>(document, 'keydown')
       .filter(event => (<HTMLElement> event.target).nodeName !== 'INPUT')
+      .filter(event => (<HTMLElement> event.target).nodeName !== 'TEXTAREA')
```

Comment:
@YinanDanielZhou I think we should do a more fundamental fix of this problem. We should try to detect the current focused element, and if it's within the jointjs element, then delete it. Try to do some research on HTML activeElement. For this PR we can merge it, but we should do another PR that changes this implementation using activeElement

Reply:
okay will do

Reply:
Raised an issue #788